diff --git a/cpp/ycm/ClangCompleter/TranslationUnit.cpp b/cpp/ycm/ClangCompleter/TranslationUnit.cpp
index d63f243..72417b3 100644
--- a/cpp/ycm/ClangCompleter/TranslationUnit.cpp
+++ b/cpp/ycm/ClangCompleter/TranslationUnit.cpp
@@ -32,7 +32,7 @@ namespace {
 
 unsigned EditingOptions() {
   return CXTranslationUnit_DetailedPreprocessingRecord |
-         CXTranslationUnit_Incomplete |
+         // CXTranslationUnit_Incomplete |
          CXTranslationUnit_IncludeBriefCommentsInCodeCompletion |
          CXTranslationUnit_CreatePreambleOnFirstParse |
          clang_defaultEditingTranslationUnitOptions();
